### ISSUE TITLE
Keep track of number of wins

### ISSUE THREAD

**Issue by surajkadapa** [acmpesuecc/rock_paper_scissor#2]
The circuit can only track once who has won, design a circuit to keep track of the number of times a player has won and a button to register the inputs to display the result and also a reset button for the scores to be 0. \
(Hint: Use a 7-segment display)

**Comment by Alley2717** [comment]
hey can I work on this

**Comment by PandaTGOS** [comment]
Can I get assigned to this ?

**Comment by surajkadapa** [comment]
@Alley2717 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by AdiXgit** [comment]
please assign

**Comment by HemaShree0408** [comment]
hey can i get assigned to this

**Comment by 825pranav** [comment]
hey can I be assigned to this?

**Comment by Sadhana-24** [comment]
Can I get assigned?

**Comment by Alley2717** [comment]
please assign to others, i have been assigned to two

**Comment by surajkadapa** [comment]
@PandaTGOS  Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by pranavacchu** [comment]
can i get assigned?

**Comment by dhritirai** [comment]
can i get assigned

**Comment by M-SriVidya** [comment]
Can I work on this issue?

**Comment by surajkadapa** [comment]
@AdiXgit Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by karun1763** [comment]
can i get assigned

**Comment by KunalMaverick** [comment]
is this still open?

**Comment by samarth777** [comment]
can you assign this to me

**Comment by surajkadapa** [comment]
@AdiXgit i will be reassigning it because time is out.

**Comment by surajkadapa** [comment]
@KunalMaverick Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by Sadhana-24** [comment]
can I pls get assigned for this?

**Comment by ankitkumar099** [comment]
can you assign me for this issue ??

**Comment by surajkadapa** [comment]
@sadhana235 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by Python-is-better-than-Java** [comment]
Can I be assigned this?

**Comment by Taruni-Kharia** [comment]
Can I be assigned this?

**Comment by HemaShree0408** [comment]
can i be assigned to this

**Comment by Advaith2805** [comment]
can i be assigned to this

**Comment by marshallmathers0** [comment]
can i be assigned

**Comment by 411sst** [comment]
can i get assigned if this is still open?

**Comment by surajkadapa** [comment]
@sadhana235 i am reassigning it because time is up.

**Comment by surajkadapa** [comment]
@Python-is-better-than-Java Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by ghost** [comment]
Give me access pls

**Comment by swayamchhaba** [comment]
Hey can i work on this issue

**Comment by karun1763** [comment]
can i get access to this

**Comment by charan2308** [comment]
@karun1763 you have a time limit of 1 hr

**Comment by RitwikAg120** [comment]
assign me

**Comment by AnshulBanda** [comment]
can i be assigned this?
